src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -208,29 +213,33 @@ class LazyMobXTableStore<T> {
     @computed public get downloadData()
```

Comment:
@adamabeshouse I think it is not possible to move all the changes I made to the download functions: with this extra loop, we are creating new 'extra' rows (one row per every sample instead of one row per every mutation/alteration). Currently, the `push` function only adds one string to the row, according to the order of the columns. Therefore, if I want to maintain this 'extra' rows but remove the extra loop I introduced, I could make the `download` functions to return arrays instead of strings. However, these arrays should be parsed differently, and this should be done in LazyMobXTable, and I believe that's not what it is desired.